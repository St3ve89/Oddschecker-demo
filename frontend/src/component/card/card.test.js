import { Card } from '.';
import renderer from 'react-test-renderer';

test('renders Card component', () => {
  const card = renderer.create(<Card />).toJSON();
  expect(card).toMatchSnapshot();
});
