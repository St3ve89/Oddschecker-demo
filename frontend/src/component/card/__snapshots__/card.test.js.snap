// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Card component 1`] = `
<div
  className="cardstyle__CustomCard-sc-1xh6lcz-0 gKulnT"
/>
`;
